requester/requester.go

```diff
@@ -29,9 +29,39 @@ import (
 	"sync"
 	"time"
 
+	"github.com/vkuznet/x509proxy"
 	"golang.org/x/net/http2"
 )
 
+// Return client X509 certificates
+func Certs() (tls_certs []tls.Certificate) {
```

Comment:
Move this to the main package. Add a Certs field to requester.Work.

We don't have to export this.

func certs() ([]tls.Certificate, error)

Don't name the return value. Named return values are highly discouraged if it is not for self documentation. In this case, it is clear from the type what the return value is.